microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -18,12 +18,21 @@ In addition to loading remote content, content can also be loaded locally into W
 
 These approaches are described below.
 
```

Comment:
DR's question 2 from main page of PR:

tbd

Reply:
I entered some possible lead-ins inline in the Markdown listing, as a Suggested Change comment, so the present conversation can probably be Resolved.